rest_framework/views.py

```diff
@@ -356,7 +356,15 @@ def check_throttles(self, request):
                 throttle_durations.append(throttle.wait())
 
         if throttle_durations:
-            self.throttled(request, max(throttle_durations))
+            # Filter out `None` values which may happen in case of config / rate
+            # changes, see #1438
+            durations = [
+                duration for duration in throttle_durations
+                if duration is not None
+            ]
+
+            duration = max(durations) if durations else None
```

Comment:
I'm pretty sure you can use `max(durations, default=None)` here.

https://docs.python.org/3/library/functions.html#max

It's a little known feature of the `max` function for handling empty iterables.

Reply:
That's certainly true. I'm not sure which is more explicit, as you mention the `default=None` feature is quite unknown, to be fair - I just learned about that myself. I'll leave it to other reviewers to chime in here.

Reply:
Agree with @kevin-brown.

Reply:
Updated in f72e4083